dask/dataframe/groupby.py

```diff
@@ -1118,7 +1146,8 @@ def _finalize_mean(df, sum_column, count_column):
     return df[sum_column] / df[count_column]
 
 
-def _finalize_var(df, count_column, sum_column, sum2_column, ddof=1):
+def _finalize_var(df, count_column, sum_column, sum2_column, **kwargs):
+    ddof = kwargs.get("ddof", 1)
```

Comment:
Yes, this is one of my TODOs and a reason this PR is a draft. I have to see what else I need to pass down, and figure out how to handle the rest. I think I need to see what Pandas does if I pass unexpected parameters, and match the behavior.

Reply:
`numeric_only` is a tricky one, as pandas behavior is changing. See https://github.com/dask/dask/issues/9471#issuecomment-1341961696 . For now we have been filtering the warnings that pandas raises, see this PR https://github.com/dask/dask/pull/9496 . 

@jrbourbeau I can't recall where we are in the discussion of this kwarg.

Reply:
@ncclementi Thank you, I looked over the PRs, and it looks to me that `numeric_only` would have to be tackled separately, since it would be a pretty large change, and there's no clearly defined path yet.